dd-trace-api/src/main/java/datadog/trace/api/ConfigDefaults.java

```diff
@@ -101,6 +102,16 @@ public final class ConfigDefaults {
   static final boolean DEFAULT_JMX_FETCH_MULTIPLE_RUNTIME_SERVICES_ENABLED = false;
   static final int DEFAULT_JMX_FETCH_MULTIPLE_RUNTIME_SERVICES_LIMIT = 10;
 
+  static final boolean DEFAULT_METRICS_OTEL_ENABLED = false;
+  static final OtelConfig.Exporter DEFAULT_OTEL_METRICS_EXPORTER = OtelConfig.Exporter.OTLP;
```

Comment:
```suggestion
  static final Exporter DEFAULT_OTEL_METRICS_EXPORTER = Exporter.OTLP;
```

With the edited imports above, we can simplify this line.